packages/synchro-charts/src/components/charts/sc-webgl-base-chart/UnsupportedDataTypeStatus.tsx

```diff
@@ -0,0 +1,21 @@
+import { h } from '@stencil/core';
+import { DataType } from '../../../utils/dataConstants';
+
+/**
+ * Provide messaging to clarify that the data/stream has invalid DataType(s)
+ */
+export const UnsupportedDataTypeStatus = ({ supportedDataTypes }: { supportedDataTypes: DataType[] }) => (
+  <div
+    id="unsupported-data-type-error"
+    class="unsupported-data-type-status"
+    style={{ padding: '24px', background: 'white', outline: 'solid 1px black' }}
+  >
+    <h3>Unable to render your data</h3>
```

Comment:
need to support internationalization via message overrides - check out how the "EmptyStatus" component does this.